addons/popochiu/popochiu_plugin.gd

```diff
@@ -53,36 +51,33 @@ func _enter_tree() -> void:
 	# Good morning, starshine. The Earth says hello.
 	prints(ES)
 	prints(EN)
-	print_rich('[wave]%s[/wave]' % SYMBOL)
+	print_rich("[wave]%s[/wave]" % SYMBOL)
```

Comment:
At this point I'm going to ask: should we always prefer double over single quotes for strings in the source code?

Reply:
I prefer double because it makes it easier to create messages that have apostrophes. Do you prefer single quotes?